src/psyclone/psyir/nodes/call.py

```diff
@@ -233,7 +232,7 @@ def replace_named_arg(self, existing_name, arg):
                 f"in 'insert_named_arg' in the 'Call' node was not found "
                 f"in the existing arguments.")
         self.children[index] = arg
-        self._argument_names[index] = (id(arg), named_arg[1])
+        self._argument_names[index] = (id(arg), existing_name)
```

Comment:
This method no longer does what it says. It used to  replace one named argument with another. Now it replaces a named argument with an argument and keeps the original name irrespective of whether the replacing argument is named or not. Could you please reflect this change in the description. Is it also an error if the new argument is a named argument as we just throw the name away????

Reply:
I don't think that's what it did. We just provided a new node also before, never a new name. I just changed name_arg[1] to existing_name because they already matched in the for loop and named_arg was producing a pylint error for being used outside the loop were it is defined.

> Is it also an error if the new argument is a named argument as we just throw the name away????

No, that's the id of the node that is being replaced.

That said, I modified the docstring to make this clear and also updated the equivalent Operation node.